src/main/java/org/alfresco/repo/search/impl/querymodel/impl/db/AspectSupport.java

```diff
@@ -33,18 +33,23 @@
 import org.alfresco.repo.domain.node.NodeDAO;
 import org.alfresco.repo.domain.qname.QNameDAO;
 import org.alfresco.repo.search.impl.querymodel.Argument;
-import org.alfresco.repo.search.impl.querymodel.FunctionEvaluationContext;
+import org.alfresco.repo.search.impl.querymodel.FunctionEvaluationContext;
+import org.alfresco.repo.search.impl.querymodel.JoinType;
 import org.alfresco.repo.tenant.TenantService;
 import org.alfresco.service.cmr.dictionary.DictionaryService;
 import org.alfresco.service.namespace.NamespaceService;
-import org.alfresco.service.namespace.QName;
+import org.alfresco.service.namespace.QName;
+
+import com.sun.star.lang.IllegalArgumentException;

```

Comment:
did you mean to import this instead of java.lang.IllegalArgumentException ?

Reply:
This was fixed in 9142625fd883c184d9850e52cf6254d3360b5e91   ??